ci-operator/config/openshift/origin-aggregated-logging/master.json

```diff
@@ -52,12 +52,6 @@
       "dockerfile_path": "Dockerfile.centos7",
       "context_dir": "curator/"
     },
-    {
```

Comment:
Only for master branch?

Reply:
Yes.  we removed its use for 3.11 and replaced it with the oauth-proxy.  I am uncertain if any of this percolates to the earlier releases but I'm assuming no since the name of the file is 'master'